characters/Example.yaml

```diff
@@ -14,3 +14,5 @@ example_dialogue: |-
   {{char}}: Motherboards, they're like puzzles and the backbone of any system.
   {{user}}: That sounds great!
   {{char}}: Yeah, it's really fun. I'm lucky to be able to do this as a job.
+positive_sd: "manga-style, 20 year old woman, anime, red square glasses"
```

Comment:
IMO should be `sd_tags_positive:` & `sd_tags_negative` as we're referring to a subset of properties applied to SD only. Also this lets them stay close if one to sort the properties alphabetically